### ISSUE TITLE
A11Y: Widget with role of menu must have at least one tabbable element.

### ISSUE THREAD

**Issue by bjliuyi** [IBM/carbon-components-react#1545]
===== ENVIRONMENT =====
Test Tool Type: DAP
IBM CI 162 Checkpoint Violation: 2.1.1 Keyboard (https://www.ibm.com/able/guidelines/ci162/keyboard.html)
IBM Guidance: https://aat.w3ibm.mybluemix.net/token/54722859-73c2-4646-b1dc-fbee896e9ef9/cba2a01d-0221-451b-a29d-54d63bb7badd/archives/2018OctDeploy//doc/w3/help/en-US/idhi_accessibility_check_g1177.html

===== DEFECT DESCRIPTION =====
The rule detected a WAI-ARIA widget without at least one tab stop.
Focusable components should receive focus in an order that preserves meaning and operability. Certain interactive HTML elements, such as links and form controls, are keyboard focusable by default. However, WAI-ARIA (ARIA) “widgets” are not natively reachable by keyboard, and the author must address this limitation. It is best achieved by including these widgets in the tab sequence, or by making them accessible from a component that is in the tab sequence (e.g., part of a composite component).
ARIA widgets are custom user interface objects given an explicit ARIA role that matches its intended purpose. These “roles” can be used to build standalone user interface widgets, or they might be used to create individual parts of a composite widget (e.g., role="button", role="checkbox", role="comobox", etc.).
![image](https://user-images.githubusercontent.com/34873342/48245401-93d60600-e425-11e8-8cd7-85e10a970352.png)

**Comment by asudoh** [comment]
Hi 👋 could you see what component you hit this problem with? Thanks!

**Comment by bjliuyi** [comment]
this issue is fixed now. Close
